content/applications/productivity/discuss/overview/plan_activities.rst

```diff
@@ -1,64 +1,198 @@
+      clock icon may be replaced by a :guilabel:`📞 (phone)` or an :guilabel:`✉️ (envelope)` icon.
+
+    .. image:: plan_activities/schedule-list-activity.png
+       :align: center
+       :alt: List view of the CRM pipeline and the option to schedule an activity.
+
+   .. tab:: Activity view
+
+      Select a record on which to schedule an activity. Move across the row to find the desired
+      activity type, then click the :guilabel:`＋ (plus sign)`.
+
+    .. image:: plan_activities/schedule-activity-view.png
+       :align: center
+       :alt: Activity view of the CRM pipeline and the option to schedule an activity.
+
+View scheduled activities
+=========================
+
+To view scheduled activities, click the :guilabel:`🕘 (clock)` icon on the menu bar. Doing so opens
+the :guilabel:`Activities menu`. Activities are separated out by the application where they were
+created, and by deadline. Select an icon to see the activities for that specific application.
+
+.. image:: plan_activities/activities-menu.png
+   :align: center
+   :alt: View of CRM leads page emphasizing the activities menu.
+
+This opens the :guilabel:`My Activities` list. Click on the icon in the :guilabel:`Next Activity`
+column to view details on the scheduled activity, or to schedule an additional activity. To delay
+the current activity's deadline by a week, click :guilabel:`Snooze 7d`.
+
+.. image:: plan_activities/my-activities-list.png
+   :align: center
+   :alt: View of CRM leads page emphasizing the activities menu.
+
+To filter the list by overdue activities, use the :guilabel:`Filters` dropdown below the search bar.
 
-Set your activity types
------------------------
+Configure activity types
+========================
 
-A number of activities types are available by default in Odoo (call, email, meeting, etc.). However,
-you can set new ones going to :menuselection:`Settings --> Activity types`.
+To configure the types of activities in the database, go to :menuselection:`Settings application -->
+Activity Types`.
 
-.. image:: plan_activities/settings_activities_types.png
+.. image:: plan_activities/settings-activities-types.png
    :align: center
-   :height: 300
-   :alt: View of the settings page emphasizing the menu activity types for Odoo Discuss
+   :alt: View of the settings page emphasizing the menu activity types.
 
 .. tip::
-   If you need to create an activity type with an available calendar, make sure to create
-   it with an *Action to Perform* set as *Meeting*.
+   Individual applications will have a list of the *Activity Types* for that application. For
+   example, to view and edit the activities available for the *CRM* application, go to
+   :menuselection:`CRM --> Configuration --> Activity Types`.
 
-Recommend next activities
--------------------------
+To edit an existing activity type, select it from the list, then click :guilabel:`Edit`. To create a
+new activity type, click :guilabel:`Create`.
```

Comment:
If you're forward porting this doc-- may be helpful to word-- If applicable, then click edit.....